pyaqueduct/api.py

```diff
@@ -179,3 +180,53 @@ def get_extensions(self) -> List[Extension]:
             )
             for extension_data in self._client.get_extensions()
         ]
+
+    @validate_call
+    def get_task(
+        self,
+        task_id
+    ) -> Task:
+        """Get task by passing task_id.
+
+        Returns:
+            Task object
+        """
+        return Task(
+            client=self._client,
+            task_id=task_id
```

Comment:
Not sure, below in definition of `Task` class has a `uuid` field. Is it task id?

Reply:
The definition of class has `uuid` field, but the constructor has parameter named `task_id`. It'd be a good idea to have both as same, as it'll only create confusion that they pass `task_id` while initialising, but that ends up to have no value in object.

I'mm replacing the name of parameter `task_id` with `uuid`. Thanks...